Arrowgene.Ddon.Shared/AssetReader/EnemySpawnAssetDeserializer.cs

```diff
@@ -107,22 +115,33 @@ public EnemySpawnAsset ReadPath(string path)
                     Experience = row[enemySchemaIndexes["Experience"]].GetUInt32(),
                 };
 
-                //checking if the file has spawntime, if yes we convert the time and pass it along to enemy.cs
-                if (enemySchemaIndexes.ContainsKey("SpawnTime"))
-                    {
-                        string SpawnTimeGet = row[enemySchemaIndexes["SpawnTime"]].GetString();
-                        ConvertSpawnTimeToMilliseconds(SpawnTimeGet, out long start, out long end);
-                        enemy.SpawnTimeStart = start;
-                        enemy.SpawnTimeEnd = end; 
-                    }
-                    else
-                    {
-                        // if no, we define it to the "allday" spawn time range and pass this along to the enemy.cs instead.
-                        ConvertSpawnTimeToMilliseconds("00:00,23:59", out long start, out long end);
-                        enemy.SpawnTimeStart = start;
-                        enemy.SpawnTimeEnd = end;
-                    }
-                    
+                // checking if the file has spawntime, if yes we convert the time and pass it along to enemy.cs
+                if(enemySchemaIndexes.ContainsKey("SpawnTime"))
+                {
+                    string SpawnTimeGet = row[enemySchemaIndexes["SpawnTime"]].GetString();
+                    ConvertSpawnTimeToMilliseconds(SpawnTimeGet, out long start, out long end);
+                    enemy.SpawnTimeStart = start;
+                    enemy.SpawnTimeEnd = end;
+                }
+                else
+                {
+                    // if no, we define it to the "allday" spawn time range and pass this along to the enemy.cs instead.
+                    ConvertSpawnTimeToMilliseconds("00:00,23:59", out long start, out long end);
+                    enemy.SpawnTimeStart = start;
+                    enemy.SpawnTimeEnd = end;
+                }
+
+                // check if the file has PPDrop.
+                if(enemySchemaIndexes.ContainsKey("PPDrop"))
+                {
+                    // If yes, get it as a uint32.
+                    enemy.PPDrop = row[enemySchemaIndexes["PPDrop"]].GetUint32();
+                }
+                else
+                {
+                    // If no, set the value to its experience / EXP_PER_PP.
+                    PPDrop = row[enemySchemaIndexes["Experience"]].GetUint32() / EXP_PER_PP;
```

Comment:
Shouldn't this be enemy.PPDrop? Also, at this point in the code you could do enemy.Experience / EXP_PER_PP instead of parsing the value again